install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -4,6 +4,9 @@
 # For example: 1.7, 1.8, 1.9
 # upgradeCompatibility: '1.8'
 
+# -- String to override release namespace for all resources
```

Comment:
Can we add a bit more of context here? e.g. mention that it is useful for tools that are unable to pass other Helm flags than values like argo-cd.

Reply:
Thanks @pr0PM! Are you going to replace more `.Release.Namespace` in `install/kubernetes/cilium`?